svelte/src/components/markdown/markdown.svelte

```diff
@@ -28,8 +28,10 @@
   // TODO: support gitlab too
   const changeSrcIfNeeded = (element: HTMLElement | Element) => {
     const src = element.getAttribute("src");
-    if (src?.startsWith("/")) {
-      element.setAttribute("src", `https://raw.githubusercontent.com${src}`);
+    const isRelativePath = src?.match(/^[^/]+\/[^/].*$|^\/[^/].*$/);
+    if (isRelativePath) {
+      const url = new URL(src!, "https://github.com");
```

Comment:
should this be `const url = new URL(src, “https://github.com”);

Reply:
i fixed that. it was possible for src to null